desktop/apps/auction2/components/filter_sort/index.js

```diff
@@ -0,0 +1,54 @@
+import { default as React, PropTypes } from 'react';
+import { connect } from 'react-redux'
+import _ from 'underscore'
+import { updateSort } from '../../actions'
+
+function FilterSort({ sortMap, filterParams, updateSortAction }) {
+  const selectedSort = filterParams.sort
+  return (
+    <div className='auction2-filter-sort'>
+      <span className='auction2-filter-sort__label bordered-pulldown-label'>Sort by:</span>
+      <div className='bordered-pulldown'>
+        <a className='bordered-pulldown-toggle' href='#'>
+          <span className='bordered-pulldown-text'>{ sortMap[selectedSort] }</span>
+          <div className='bordered-pulldown-toggle-caret'>
+            <span className='caret'></span>
+          </div>
+        </a>
+        <div className='bordered-pulldown-options'>
+          {
+            _.map(sortMap, (sortName, sort) => {
```

Comment:
I may be missing something obvious, but it doesn't seem that calling `.map` on objects is native to JS (at least, when I try to run `sortMap.map` I get:

![image](https://cloud.githubusercontent.com/assets/2081340/23715209/dc52f266-03f9-11e7-83a1-09495206ef7b.png)